texar/utils/beam_search.py

```diff
@@ -120,15 +123,15 @@ def compute_batch_indices(batch_size: int, beam_size: int) -> torch.LongTensor:
     return batch_pos
 
 
-def compute_topk_scores_and_seq(
-    sequences: torch.LongTensor,
-    scores: torch.Tensor,
-    scores_to_gather: torch.Tensor,
-    flags: torch.ByteTensor,
-    beam_size: int,
-    batch_size: int,
-    states_to_gather: Optional[State] = None,
-) -> Tuple[torch.LongTensor, torch.Tensor, torch.ByteTensor, Optional[State]]:
+def compute_topk_scores_and_seq(sequences: torch.LongTensor,
```

Comment:
Code format changes like this are unnecessary and creates irrelevant clutter in code diffs. Please consider reverting these changes.